k8s/namespaces/family-public-law/fpl-case-service/demo.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: fpl-case-service
+spec:
+  values:
+    java:
+      image: hmctspublic.azurecr.io/fpl/case-service:prod-5a4b25f5
+      ingressHost: fpl-case-service-demo.service.core-compute-demo.internal
+      environment:
+        GATEWAY_URL: https://gateway-ccd.demo.platform.hmcts.net
+        AUTH_IDAM_CLIENT_BASEURL: https://idam-api.demo.platform.hmcts.net
+        TESTING_SUPPORT_ENABLED: true
+      aadIdentityName: family-public-law
```

Comment:
You don't need to override this as its defaulted in https://github.com/hmcts/fpl-ccd-configuration/blob/master/charts/fpl-case-service/values.yaml#L11 

```suggestion
 
```

Reply:
removed